sdk/digitaltwins/digitaltwins/api-extractor.json

```
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "mainEntryPointFilePath": "types/src/index.d.ts",
+  "docModel": {
+    "enabled": false
```

this should be true now

```suggestion
    "enabled": true
```

---

Changed

---
In reply to: [470195384](https://github.com/Azure/azure-sdk-for-js/pull/10560#discussion_r470195384) [](ancestors = 470195384)